### DISCUSSION TITLE
How to update

### DISCUSSION THREAD

**Discussion by win98se** [namazso/SecureUxTheme#149] [Q&A] [answered]
Now v2.2.0 is out, how do I update from v2.1.2?
Precisely, do I launch ThemeTool as administrator, click on `Uninstall` and then `Install`? Or I can totally skip the above steps and patch the themes as usual?

**Comment by namazso** [comment, marked as answer]
If everything is working, you don't really *need* to. Anyways, just Install should work if you really want to.
